src/discovery/azure/Akka.Discovery.Azure/AkkaHostingExtensions.cs

```diff
@@ -240,8 +240,12 @@ public static class AkkaHostingExtensions
             AkkaDiscoveryOptions options)
         {
             options.Apply(builder);
-
-            builder.AddHocon(AzureServiceDiscovery.DefaultConfig, HoconAddMode.Append);
+            
+            // force start the module
+            builder.AddStartup((system, registry) =>
+            {
+                AzureDiscovery.Get(system);
```

Comment:
Why is this necessary now but not before?

Reply:
Before, the discovery module was automatically started by ClusterBootstrap during startup. Since ClusterBootstrap might not run in tandem with AzureDiscovery anymore, we need to force start it.